buildscripts/conda/mantidimaging-gui/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: mantidimaging
```

Comment:
Should the name here be `mantidimaging-gui`?

Reply:
I assumed not since it is still the same package. The only difference is that GUI related dependencies are included and the app entry point is defined. Both packages actually include the GUI code, but it would only work in this one.

I guess a better way to do this would be to have completely separate packages for the core and GUI.